tests/cli/test_units.py

```diff
@@ -43,19 +46,52 @@ def tearDown(self):
         shutil.rmtree(self.tmproot)
 
     def test_is_composite_member(self):
```

Comment:
we might want to drop this all together. `is_composite_member` is not part of any public API and not used any more within the CLI